### PR TITLE
 Fix avocado:core:sysinfo 

### PR DESCRIPTION
**Pull request by PraveenPenguin**
1- fixed pep8 issue 
2- avocado crashed if user provide non exsting profiler
this patch address that , if profiler not avilable than
framework  handle gracefully

### CODE REVIEW

File: avocado/core/sysinfo.py [hunk 1]
```diff
@@ -214,8 +215,8 @@ def stop(self):
             process.kill_process_tree(self.pipe.pid)
             retcode = self.pipe.wait()
         else:
-            log.error("Daemon process '%s' (pid %d) terminated abnormally (code %d)",
-                      self.cmd, self.pipe.pid, retcode)
+            log.error("Daemon process '%s' (pid %d) terminated abnormally "
+                      "(code %d)", self.cmd, self.pipe.pid, retcode)
```
Comment (clebergnu) [inline avocado/core/sysinfo.py]: There are no issues with these line disposition, but, bear in mind that you can go as far as 79 chars, which would result in:

```python
# following line is column numbers
#2345678901234567890123456789012345678901234567890123456789012345678901234567890
                log.error("Daemon process '%s' (pid %d) terminated abnormally "
                          "(code %d)", self.cmd, self.pipe.pid, retcode)
```

File: avocado/core/sysinfo.py [hunk 2]
```diff
@@ -186,6 +186,7 @@ class Daemon(Command):
     :param logf: Basename of the file where output is logged (optional).
     :param compress_logf: Whether to compress the output of the command.
     """
+    pipe = None
```
[the remark this answers is not in the document]
Comment (PraveenPenguin) [reply avocado/core/sysinfo.py]: @clebergnu sure make sense will adopt

File: avocado/core/sysinfo.py [hunk 3]
```diff
@@ -202,22 +203,27 @@ def run(self, logdir):
...
         stdout = open(logf_path, "w")
-        self.pipe = subprocess.Popen(shlex.split(self.cmd), stdin=stdin,
-                                     stdout=stdout, stderr=subprocess.STDOUT,
-                                     shell=False, env=env)
+        try:
+            self.pipe = subprocess.Popen(shlex.split(self.cmd), stdin=stdin,
+                                         stdout=stdout, stderr=subprocess.STDOUT,
+                                         shell=False, env=env)
+        except OSError:
+            log.debug("Not logging as %s command not able to run " % self.cmd)
```
[the remark this answers is not in the document]
Comment (PraveenPenguin) [reply avocado/core/sysinfo.py]: sure

File: avocado/core/sysinfo.py (hunk 1 above)
Comment (PraveenPenguin) [reply avocado/core/sysinfo.py]: ah yes i miss that ,will make is pep8 complaint

**Comment by clebergnu** [comment]
@PraveenPenguin I'm not sure if the current commits address the issues I pointed out during the review.  Would you care to send an updated version (a [v2]) so that it also passes Travis?  Thanks!

**Comment by PraveenPenguin** [comment]
@clebergnu sure i will send V2 having adopted all the changes

**Comment by PraveenPenguin** [comment]
@clebergnu pused V2 #2778  please have look , unit test case i will push in some time

**Pull request closed without merging**